hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/rsql/RsqlParserValidationOracle.java

```diff
@@ -61,8 +61,11 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RsqlParserValidationOracle.class);
 
-    @Autowired
-    private TargetManagement targetManagement;
+    private final TargetQueryExecutionManagement targetQueryExecutionManagement;
+
+    public RsqlParserValidationOracle(@Autowired final TargetQueryExecutionManagement targetQueryExecutionManagement) {
```

Comment:
Within the Spring configuration you explicitly pass an instance of `TargetQueryExecutionManagement` into the constructor. So why you use `@Autowired` here?

Reply:
Just moved it from
```
public class RsqlParserValidationOracle implements RsqlValidationOracle {

    private static final Logger LOGGER = LoggerFactory.getLogger(RsqlParserValidationOracle.class);

    @Autowired
    private TargetManagement targetManagement;
...
```
to constructor injection.